cogip/robotentity.py

```diff
@@ -135,8 +135,8 @@ def set_dyn_obstacles(self, dyn_obstacles: DynObstacleList) -> None:
             p0, p1, p2, p3 = dyn_obstacle.__root__
             length = math.dist((p0.x, p0.y), (p1.x, p1.y))
             width = math.dist((p1.x, p1.y), (p2.x, p2.y))
-            pos_x = min(p0.x, p1.x) + length/2
-            pos_y = min(p0.y, p2.y) + width/2
+            pos_x = p0.x + (p2.x - p0.x)/2
+            pos_y = p0.y + (p2.y - p0.y)/2
```

Comment:
Can you add a comment about what are pos_x and pos_y ?

Reply:
pos_x and pos_y are the position of the obstacle, does it really deserve a comment?

Reply:
Done